### PR TITLE
chore: document joining/leaving governance steps

### PR DESCRIPTION
**Pull request by sofianguy**
Taking a first attempt at documenting the steps to add a new governance member and steps for when a member leaves governance. Open to suggestions!

### CODE REVIEW

File: policy/joining-governance.md
Comment (molant) [inline policy/joining-governance.md:13]:
```suggestion
1. Submit the Workflow form for "New Governance Member" in #wg-community-safety channel.
```

File: policy/joining-governance.md
Comment (molant) [inline policy/joining-governance.md:23]:
```suggestion
6. Submit a pull request to add the new member's GitHub handle under the approved Working Group in the `config.yaml` file in [.permissions repo](https://github.com/electron/.permissions/) .
```

File: policy/joining-governance.md
Comment (molant) [inline policy/joining-governance.md:17]:
```suggestion
3. An Electron GitHub owner invites the new member to the Electron Org in GitHub. New member needs to accept this to continue the process.
```

File: policy/joining-governance.md
Comment (molant) [inline policy/joining-governance.md:19]:
```suggestion
4. Request a Google account through the "Request @electronjs.org" shortcut in #access-request Slack channel.
```

Who should request the Google account? The chair of the WG?

File: policy/joining-governance.md
Comment (molant) [inline policy/joining-governance.md:21]:
```suggestion
5. Approve the Google account request to the new member.
```

File: policy/joining-governance.md
Comment (molant) [inline policy/joining-governance.md:25]:
```suggestion
7. A member of Community & Safety WG approves the PR in .permissions repo.
```

File: policy/leaving-governance.md
Comment (molant) [inline policy/leaving-governance.md:13]:
```suggestion
* Request an admin of Electron's Slack (any member of Community & Safety WG) to change the contributor to a guest account.
```

**Review by molant — CHANGES_REQUESTED** [review]
Thanks a lot @sofianguy for documenting this!

One thing I'd like to see is to clearly state who owns each step. You mention it in some of them (e.g. "Electron GitHub owner", "a member of Community & Safety"), but there are others that might not clear, especially for someone that hasn't seen this process before (like me 😅). Should it be the chair of the WG?

I've also proposed some minor changes to normalize the language a bit the language and what I believe are a couple typos, but those are more nits than anything else.

File: policy/joining-governance.md
Comment (sofianguy) [reply policy/joining-governance.md:21]: This step here isn't just an approval. The google account must be made in order for the magic to happen. I'll reword this to be more clear about it.

**Comment by sofianguy** [comment]
@molant thanks for reviewing and catching all my typos (😅 )!

I edited to add owners for each step. Let me know if this is what you had in mind.

File: policy/joining-governance.md
Comment (MarshallOfSound) [inline policy/joining-governance.md:24]:
```suggestion
    - Current owners of GSuite set up are @MarshallOfSound and @codebytere.
```

File: policy/joining-governance.md
Comment (MarshallOfSound) [inline policy/joining-governance.md:33]: Sheriff doesn't do this _yet_ but it will soon, if we leave this PR hanging for a bit it will be correct 😄

Comment (sofianguy) [reply policy/joining-governance.md:33]: @MarshallOfSound 😄 ok, lmk when that's all good!

**Review by ckerr — APPROVED** [review]

**Comment by ckerr** [comment]
approved but pending re-review from @molant of course

**Review by molant — APPROVED** [review]
Sorry for the delay, I've been in meetings all day 😓
LGTM!

**Pull request merged**
